project/build.properties

```diff
@@ -1 +1 @@
-sbt.version=0.13.15
+sbt.version=0.13.16-M1
```

Comment:
It'll be binary compatible, for sure.

Yeah it's just the overhead. Including creating windows .msi's on a Windows box, zip/tgz/rpm/deb, etc.. 0.13.16-M1 is probably only out there as a jars and git tag.